app/src/main/java/org/schabi/newpipe/util/StreamTypeUtil.java

```diff
@@ -3,22 +3,38 @@
 import org.schabi.newpipe.extractor.stream.StreamType;
 
 /**
- * Utility class for {@link org.schabi.newpipe.extractor.stream.StreamType}.
```

Comment:
IMO, this methods should be static methods in the `StreamType` enum.
But that is a task for a separate PR.